src/signalo/edge_cases/tests.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework.test import APITestCase
+
+
+class TestViewsets(APITestCase):
```

Comment:
Can you add a test case (with the same users as described below) testing the "collections" view ?